### PR TITLE
Kiyong CSS Battle

### PR DESCRIPTION
**Pull request by kiyonggg**

### CODE REVIEW

File: src/CSSBattle/1_SimplySquare/kiyong.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="box"></div>
```
Comment (shaokiat) [inline src/CSSBattle/1_SimplySquare/kiyong.html]: Didn't load in website. Might be named wrongly

Comment (kiyonggg) [reply src/CSSBattle/1_SimplySquare/kiyong.html]: resolved in commit a9937a9

**Review by shaokiat — APPROVED** [review]
LGTM

**Review by gunbux — APPROVED** [review]
lgtm! (review 2)

**Pull request merged**